scheduler/src/cook/mesos/task.clj

```diff
@@ -52,17 +54,19 @@
    2. The Cook executor command has been configured,
    3. Either :job/executor is explicitly enabled
       Or: the job is a cook-executor candidate (see cook-executor-candidate?)."
-  [job-ent {:keys [command portion retry-limit]}]
-  (and (not (use-custom-executor? job-ent))
-       command
-       (or (= :executor/cook (:job/executor job-ent))
-           (cook-executor-candidate? job-ent portion retry-limit))))
+  [job-ent]
+  (let [{:keys [command]} (-> config :settings :executor)]
+    (and (not (use-custom-executor? job-ent))
+         command
```

Comment:
Can we use `(-> config :settings :executor :command)` to avoid the explicit `let` block.

Reply:
Good call, done